rma/wizards/rma_make_picking_view.xml

```diff
@@ -134,4 +134,28 @@
         </field>
     </record>
 
+    <record id="view_rma_button_form" model="ir.ui.view">
```

Comment:
I find it quite confusing to have an override of `rma.order` view in rma_make_picking_view.xml file.
I know there is other similar cases in the rma modules but do we really want to continue this way ? Should it not go to rma_order_view.xml ? @AaronHForgeFlow

Reply:
@florian-dacosta, better to have file for model, whenever possible, in the next versions some cleanup will be done, hopefully.